### PR TITLE
[BACKLOG-17647] Dev - Add Kafka Bootstrap servers to name cluster dialog

### PR DESCRIPTION
**Pull request by pavel-sakun**
@DFieldFL

### CODE REVIEW

File: kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/NamedClusterDialogImpl.java [hunk 1]
```diff
@@ -120,67 +121,66 @@ public void dispose() {
...
     props.setLook( shell );
     shell.setImage( GUIResource.getInstance().getImageSpoon() );
+    shell.setMinimumSize( DIALOG_WIDTH, 458 );
+    shell.setText( BaseMessages.getString( PKG, "NamedClusterDialog.Shell.Title" ) );
+    FormLayout formLayout = new FormLayout();
+    formLayout.marginWidth = 15;
+    formLayout.marginHeight = 15;
+//    formLayout.marginBottom = 15;
+//    formLayout.marginLeft = 15;
+//    formLayout.marginRight = 15;
```
Comment (DFieldFL) [inline kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/NamedClusterDialogImpl.java]: Remove commented out code.

**Comment by pavel-sakun** [comment]
@DFieldFL done

**Comment by pavel-sakun** [comment]
Closing for now

**Comment by pavel-sakun** [comment]
DO NOT MERGE YET

**Pull request merged**